conf/application.conf

```diff
@@ -44,12 +44,12 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 # Custom error handler
 play.http.errorHandler = "uk.gov.hmrc.soletraderidentificationfrontend.config.ErrorHandler"
 
-play.filters.headers.contentSecurityPolicy = "script-src 'unsafe-inline' 'self' localhost:9894 localhost:12345 www.google-analytics.com www.googletagmanager.com tagmanager.google.com *.digital.nuance.com; style-src 'unsafe-inline' 'self' localhost:9894 tagmanager.google.com fonts.googleapis.com *.digital.nuance.com; img-src 'self' localhost:9894 https://ssl.gstatic.com www.gstatic.com www.google-analytics.com *.digital.nuance.com data:; font-src 'self' localhost:9894 https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com *.digital.nuance.com data:; frame-src 'self' https://www.googletagmanager.com *.digital.nuance.com;"
-
 play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.soletraderidentificationfrontend.featureswitch.core.config.FeatureSwitchingModule"
 
+play.filters.enabled += play.filters.csp.CSPFilter
```

Comment:
You have enabled Play's CSPFilter, but not defined any CSP directives. This means the service will use Google's Strict CSP Policy (which the Play documentation states is a good place to start).